config.json

```diff
@@ -1514,23 +1514,26 @@
         "uuid": "cbccd0c5-eb15-4705-9a4c-0209861f078c",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 4
+        "difficulty": 4,
+        "topics": []
       },
       {
         "slug": "kindergarten-garden",
         "name": "Kindergarten Garden",
         "uuid": "c27e4878-28a4-4637-bde2-2af681a7ff0d",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1
+        "difficulty": 1,
+        "topics": []
```

Comment:
Why is the `topics` key added? It is only really there for inspiration for the `practices`/`prerequisites` fields, but it serves no purpose from Exercism's viewpoint.

Reply:
I opened a separate [issue](https://github.com/exercism/rust/issues/1735) for me to consider what to do with this field / how to document it properly. In this PR, they are only added for simple and consistent de-/serialization.

I may or may not clean up empty topic fields when I decide what to do on the separate issue.